### PR TITLE
Small updates (various)

### PR DESCRIPTION
**Pull request by MeuhV1**
Small pedia strategy text fixes
Skips Russian UP code if Russia isn't alive
Replaced some spaces with tabs

### CODE REVIEW

File: Assets/Python/RFCUtils.py [hunk 1]
```diff
@@ -415,7 +415,7 @@ def flipUnitsInArea(self, tTopLeft, tBottomRight, iNewOwner, iOldOwner, bSkipPlo
 										continue
 									
 									if (bKillSettlers):
-										if ((unit.getUnitType() > iSettler)):
+										if ((unit.getUnitType() > iHarappanCityBuilder)):
 											self.makeUnit(unit.getUnitType(), iNewOwner, [0, 67], 1)
 									else:
 										if ((unit.getUnitType() >= iSettler)): #skip animals
```
Comment (MeuhV1) [inline Assets/Python/RFCUtils.py]: Shouldn't line 418 and 421 be swapped?

File: Assets/Python/RFCUtils.py [hunk 2]
```diff
@@ -172,16 +172,16 @@ def isMortalUnit(self, unit):
 				if (gc.getGame().getSorenRandNum(100, 'random modifier') >= 50):
 					return False	      
 		iUnitType = unit.getUnitType()
-		if (iUnitType <= con.iKhmerBallistaElephant \
```
Comment (dguenms) [inline Assets/Python/RFCUtils.py]: Good catch, but this method is a mess in general. I pushed a much cleaner implementation to develop, you might need to rebase though. Sorry about that.

File: Assets/Python/RFCUtils.py (hunk 1 above)
Comment (dguenms) [inline Assets/Python/RFCUtils.py]: Yes. Can you also use unit.isFound() and unit.isAnimal() instead?

File: Assets/Python/RFCUtils.py [hunk 3]
```diff
@@ -700,7 +700,7 @@ def pushOutGarrisons(self, tCityPlot, iOldOwner):
 				if (unit.getDomainType() == 2): #land unit
 					unit.setXYOld(tDestination[0], tDestination[1])
 				else:
-					j = j + 1
```
Comment (dguenms) [inline Assets/Python/RFCUtils.py:686]: Rhye and his C idioms :)

File: Assets/Python/RiseAndFall.py [hunk 4]
```diff
@@ -2708,10 +2708,7 @@ def handleColonialConquest(self, iPlayer):
 					break
 
 		if tSeaPlot != -1:
-			if iPlayer == iNetherlands:
```
Comment (dguenms) [inline Assets/Python/RiseAndFall.py:2711]: :+1:

File: Assets/XML/Text/Units.xml [hunk 5]
```diff
@@ -989,7 +989,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_SCOUT_STRATEGY</Tag>
-		<English>[COLOR_UNIT_TEXT]Scouts[COLOR_REVERT] is able to enter [COLOR_HIGHLIGHT_TEXT]Jungle[COLOR_REVERT], [COLOR_HIGHLIGHT_TEXT]Rainforest[COLOR_REVERT] and [COLOR_HIGHLIGHT_TEXT]Marsh[COLOR_REVERT], gain only positive bonuses from Tribal villages, can only defend and have a combat bonus against [COLOR_UNIT_TEXT]Animal Units[COLOR_REVERT].</English>
```
Comment (dguenms) [inline Assets/XML/Text/Units.xml:992]: Isn't the other verb wrong? Scouts is plural.

File: Assets/XML/Text/Units.xml [hunk 6]
```diff
@@ -844,7 +844,7 @@
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_UNIT_OTTOMAN_JANISSARY_STRATEGY</Tag>
-		<English>The [COLOR_UNIT_TEXT]Janissary[COLOR_REVERT], the Ottoman replacement for the [COLOR_UNIT_TEXT]Musketman[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Archery Units[COLOR_REVERT], [COLOR_UNIT_TEXT]Mounted Units[COLOR_REVERT] and [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</English>
+		<English>The [COLOR_UNIT_TEXT]Janissary[COLOR_REVERT], the Turkish replacement for the [COLOR_UNIT_TEXT]Musketman[COLOR_REVERT], has a combat bonus against [COLOR_UNIT_TEXT]Archery Units[COLOR_REVERT], [COLOR_UNIT_TEXT]Mounted Units[COLOR_REVERT] and [COLOR_UNIT_TEXT]Melee Units[COLOR_REVERT].</English>
```
Comment (SgtBears) [inline Assets/XML/Text/Units.xml]: What about the Ottoman Hammam?

**Comment by SgtBears** [comment]
You changed the "ottoman" to "turkish" only for the janissary. You didn't change the Hammam. :)

**Comment by MeuhV1** [comment]
I didn't see scouts was plural. Will revert it. However, I just saw "Scouts is", which is wrong.
I'll add isFound and isAnimal.
Will also change Hammam to Turkish.

**Comment by MeuhV1** [comment]
Done.

**Comment by MeuhV1** [comment]
Update:

New art Floating Gardens
Highlighted "Scientist" in Arabian Alchemist UB (sync with other strategy texts)
Fixed text error in Victory.py (Harappan Granary -> Walls)
Brazilian UU ability also works on rainforest
Removed some tabs and spaces in Plague.py

**Pull request merged**